README.md

```diff
@@ -43,12 +43,12 @@ Get started with the benchmarks:
 
 <!-- PERFORMANCE_RESULTS_START -->
 
-| Server           | Requests/sec | Latency (ms) |
-| ---------------- | -----------: | -----------: |
-| [Tailcall]       |   `2,890.68` |      `34.69` |
-| [Gqlgen]         |     `935.00` |     `115.73` |
-| [Apollo GraphQL] |     `793.37` |     `128.22` |
-| [Netflix DGS]    |     `597.39` |     `191.85` |
+| Server | Requests/sec | Latency (ms) |
+|--------:|--------------:|--------------:|
+| [Tailcall] | `6066.06` | `16.46` |
+| [Gqlgen] | `2088.87` | `49.66` |
+| [Apollo GraphQL] | `1673.16` | `59.97` |
+| [Netflix DGS] | `1444.94` | `73.29` |
```

Comment:
I've pushed the changes to use `,` as a separator, but the CI is still running https://github.com/ologbonowiwi/graphql-benchmarks/actions/runs/6902336459/job/18778839441